obs2ioda-v2/src/cxx/netcdf_attribute.cc

```diff
@@ -45,6 +45,15 @@ namespace Obs2Ioda {
         }
     }
 
+    int netcdfPutAttIntArray(
+        int netcdfID, const char *attName, const int *attValue,
+        const int attLen, const char *varName, const char *groupName
+    ) {
+        return netcdfPutAtt(
+            netcdfID, attName, attValue, varName, groupName,
+            netCDF::NcType(netCDF::ncInt), attLen
+        );
+    }
```

Comment:
Can you correct the commit history of the PR branch so the changes to support integer array attributes don't show up as part of this PR?